server/scratchy_server/model/objectId.py

```diff
@@ -0,0 +1,18 @@
+import marshmallow as ma
+import bson
+
+
+class ObjectId(ma.fields.Field):
+    """
+    Marshmallow field for :class:`bson.ObjectId`
+    """
+    def _serialize(self, value, attr, obj):
+        if value is None:
+            return None
+        return str(value)
```

Comment:
You need to create a ticket for the front team, tell them the format as changed. Ticket should mention :  
- old format and new format and request a migration
- reference this PR / ticket

Reply:
Ticket created here : #151